test/conftest.py

```diff
@@ -39,6 +39,10 @@ def __init__(self):
 
         driver = self.get_driver()
         wait = WebDriverWait(driver, timeout=20)
+        if not (BUILD_PATH / 'test.html').exists():
+            # selenium does not expose HTTP response codes
+            raise ValueError(f"{(BUILD_PATH / 'test.html').resolve()} "
+                             f"does not exist!")
```

Comment:
I think this needs to be removed also?

Reply:
Never mind.  I see this is just the warning.  :+1:

Reply:
Yes, it just aims to address the fact that when `wait.until(PyodideInited())` fails we end up with an unhelpful error message and it takes `N_tests*timeout ~ 2 hours` to complete. This detects one of reasons this may fail, and error early in that case.